src/lib.rs

```diff
@@ -949,6 +958,28 @@ pub fn setup_new_windows_system(
     }
 }
 
+/// Captures pointers on egui windows for bevy_picking
+pub fn capture_pointer_input(
+    pointers: Query<(&PointerId, &PointerLocation)>,
+    mut egui_context: Query<(Entity, &mut EguiContext, &EguiSettings)>,
+    mut output: EventWriter<PointerHits>,
+) {
+    for (pointer, location) in pointers
+        .iter()
+        .filter_map(|(i, p)| p.location.as_ref().map(|l| (i, l)))
+    {
+        if let NormalizedRenderTarget::Window(id) = location.target {
+            if let Ok((entity, mut ctx, settings)) = egui_context.get_mut(id.entity()) {
+                if settings.capture_pointer_input && ctx.get_mut().wants_pointer_input() {
+                    let entry = (entity, HitData::new(entity, 0.0, None, None));
+                    let order = 1_000_000f32;
```

Comment:
I'd recommend making this a public const, sometimes you need to order other backends relative to this.